modules/edu.gemini.seqexec.engine/src/main/scala/edu/gemini/seqexec/engine/model.scala

```diff
@@ -222,7 +257,12 @@ object Queue {
 /**
   * A list of Steps grouped by target and instrument.
   */
-case class Sequence[A](id: String, steps: NonEmptyList[Step[A]])
+case class Sequence[A](
+  id: String,
+  steps: NonEmptyList[Step[A]],
+  // Unrooted executions
```

Comment:
BTW, the concept of `Execution` shouldn't show up outside of the Engine. It internally represents a set of `Action`s that need to be executed in parallel.